src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/NoBlockRewardKeepTipFiller.yml

```diff
@@ -0,0 +1,138 @@
+      code: |
+         :yul {
+            // Inside block n we can see the balance at the end of block n-1,
+            // because the end of block processing hasn't happened yet
+            let miner := 0xBA5E000000000000000000000000000000000000
+            let blockReward := 2000000000000000000  // Block reward, 2 ETH (in wei)
+            let lastBalance := 0xFFFF   // Storage for last balance
+            let minerBalance := balance(miner)
+
+            if eq(number(), 1) { blockReward := 0 }
+
+            sstore(sub(number(),1),   // The miner gets paid after the block,
+                                      // so in block n we see the result of block n-1
+               minerBalance
+            )
+
+            // Update the balance. If we got any data, zero it so we won't have
+            // to keep track for the expect: section
+            if eq(calldatasize(), 0) { sstore(lastBalance, minerBalance) }
+            if gt(calldatasize(), 0) { sstore(lastBalance, 0) }
+         }
+      storage: {}
+
+  blocks:
+  # Block 1
+  - transactions:
+    - data: ''
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+  # Block 2
+  - transactions:
+    - data: ''
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+  # Block 3, maxPriorityFeePerGas=1
+  - transactions:
+    - data: ''
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+    # This transaction gives the miner 21003 wei because it uses 21003 gas
+    # 21k for the transaction, and 3 for the push that AA..AA does
+    - data: ''
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0x01
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+      value: 0x0
+
+    # Clear 0xFFFF so we won't have to put it in the expect: section.
+  - transactions:
+    - data: :raw 0x60A7
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+  expect:
+  - network:
+    - "Merge"
+    result:
+      # Change in miner balance, minus the block reward
```

Comment:
block reward is 0 in Merge, how come?

Reply:
The block rewards are not given out but the miner tips are still being given to the beneficiary address. Possibly caused in the blockchain PoS rules.